third_party/docfx/templates/devsite/token.json

```diff
@@ -31,7 +31,7 @@
   "overrides": "Overrides",
   "implements": "Implements",
   "remarks": "Remarks",
-  "examples": "Examples",
+  "example": "Example",

```

Comment:
Some of them really do have multiple examples. So, we'd need to do some sort of detection of if there is only one element (e.g. check if a second element exists).

The other headers (remarks, overrides, implements, parameters, etc.) are all plural, even if the section only has one element.

Reply:
Seeing that this can be done for all of the other headers, I think we should do this for all other headers as well.

For now I'll move forward and try to work this bit out. Hopefully it's not too much JS trickery :S

Reply:
Perhaps we should update the header content in a separate PR from the header levels?

Reply:
Sounds good. I'll leave this out for now for this PR (but almost got it working!)

Reply:
Looks like this change is still included.